portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +31,69 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  /**
+   * Takes in a query object and places its content in a json formatted string
+   * and returns it.
+   */
+  private String getJson(Query commentQuery){
+    String returnStr = "{";
```

Comment:
+= for Strings here is fine since it's a pretty small case and not performance sensitive. 

But just FYI usually it's a better idea to user a StringBuilder in these types of situations (concatenation in a for-loop) because of performance reasons. See: https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java